python-sdk/python_sdk_custom_docker_demo/main.py

```diff
@@ -0,0 +1,55 @@
+import disco
+from disco.core.constants import JobStatus
+import pathlib
+from disco.docker_image import DockerImage
+
+#we will timeout if the job ran for over 5 minutes
+#this is up to your application and change it accordingly
+LIVE_TESTS_TIMEOUT_SECONDS = 300
+
+DISCO_EMAIL = 'ray_ifx@dis.co'
+DISCO_PASSWORD = 'impact123'
+
+disco.set_credentials(DISCO_EMAIL, DISCO_PASSWORD, save_to_config=True)
+
+#list of docker images we got
+i = 0
+docker_images = DockerImage.list_docker_images()
+for docker_image in docker_images:
+    print("================================================================")
+    print("Selection Number: "+str(i))
+    print("ID: "+str(docker_image.id))
+    print("Name: "+str(docker_image.name))
+    print("Is Active: "+str(docker_image.is_active))
+    print("Repository Type: "+str(docker_image.repository_type))
+    print("================================================================")
+    i = i + 1
+
+print("Please Select Your Docker Image. Enter the number 0, 1, 2, 3, ...")
+selection = input()
+docker_id = DockerImage.list_docker_images()[int(selection)].id
+
+
+#this will upload a simple server script and runs the tasks
+script_id = disco.upload_file('server.py', pathlib.Path('./server.py'))
```

Comment:
a comment for this line a generally for all examples. For the reader to better understand what the parameters are, use named parameters. i.e - (file_name='server.py',script_path='...') I'm not sure it's the right names i used here (pretty sure it's not) but you will see them when you open the parentheses.